libraries/Ethernet/src/EthernetClient.cpp

```diff
@@ -173,5 +180,5 @@ bool EthernetClient::operator==(const EthernetClient& rhs) {
 }
 
 uint8_t EthernetClient::getSocketNumber() {
-  return _sock;
+  return 0;
 }
```

Comment:
why does getSocketNumber always return 0  - is this a limitation of the implementation